src/_locales/es-ES.json

```diff
@@ -0,0 +1,96 @@
+{
+  "audio-device-communications": "Sistema de comunicaciones por defecto",
+  "audio-device-default": "Dispositivo por defecto",
+  "audio-device-unknown": "Dispositivo Desconocido",
+
+  "button-test-ok": "Aceptar",
+  "button-text-close": "Cerrar",
+  "button-text-dont-tell-me-again": "No recordar más",
+  "button-text-lets-do-this": "Hagámoslo!!",
+  "button-text-lets-go": "Vamos!!",
+  "button-text-not-now": "Ahora mismo no",
+
+  "label-about": "Acerca de",
+  "label-desktop-settings": "Ajustes de escritorio",
+  "label-donate": "Donar",
+  "label-help": "Ayuda",
+  "label-issues": "Problemas",
+  "label-learn-more": "Aprender más",
+  "label-loading-devices": "Cargando Dispositivos...",
+  "label-pause-after-song": "Pausar después de esta canción",
+  "label-quit": "Salir",
+  "label-show-lyrics": "Mostrar Letras (Beta)",
+  "label-unknown-artist": "Artista desconocido",
+  "label-unknown-song": "Canción desconocida ",
+  "label-version": "Versión",
+
+  "lastfm-login-authorized": "Autorizado",
+  "lastfm-login-authorizing": "Autorización en progreso",
+  "lastfm-login-button-text": "Accediendo a Last.FM",
+  "lastfm-login-error": "Parece que has dejado el proceso de autorización, si es un error, prueba a autorizar de nuevo",
+  "lastfm-login-not-authorized": "No autorizado",
+  "lastfm-logout-button-text": "Saliendo de Last.FM",
+
+  "lyrics-failed-message": "No hemos enconrado la letra de esta canción",
+  "lyrics-loading-message": "Cargando letra...",
+  "lyrics-no-song-message": "No hay canción sonando ahora",
+
+  "message-pausing-after-song": "Pausando después de esta canción.",
+  "message-pausing-after-song-button": "No pausar después de esta canción",
+  "message-uncaught-error": "Un error inadvertido ha ocurrido dentro de GPMDP.",
```

Comment:
"inadvertido" is not quite a right translation for uncaght, "inesperado" is a better choice, as an uncaught error is usually an unexpected one. Also "dentro de" is a bit wordy, "en" sounds better.